front/src/pages/settings/SettingsWorkspaceMembers.tsx

```diff
@@ -119,6 +126,21 @@ export function SettingsWorkspaceMembers() {
           ))}
         </Section>
       </StyledContainer>
+      <ConfirmationModal
+        isOpen={isConfirmationModalOpen}
+        setIsOpen={setIsConfirmationModalOpen}
+        title="Account Deletion"
+        subtitle={
+          <>
+            This action cannot be undone. This will permanently delete this user
+            and remove him from all his assignements.
```

Comment:
This action cannot be undone. This will permanently delete this user and remove them from all their assignements.